samples/java-customer-registry/src/main/java/customer/CustomerValueEntityHandler.java

```diff
@@ -56,36 +54,43 @@
   }
 
   @Override
-  public ValueEntityBase.Effect<Any> handleCommand(
+  public ValueEntityBase.Effect<? extends GeneratedMessageV3> handleCommand(
       Any command, Any state, CommandContext<Any> context) throws Throwable {
+
     AdaptedCommandContext<CustomerDomain.CustomerState> adaptedContext =
         new AdaptedCommandContext(context, anySupport);
-    try {
-      CustomerDomain.CustomerState parsedState =
-          CustomerDomain.CustomerState.parseFrom(state.getValue());
-      // TODO we used to support passing in the command as Jackson-parsed model object
-      // or as ScalaPB class as well. With this change we tie ourselves to Java protobuf.
-      ValueEntityBase.Effect<? extends GeneratedMessageV3> effect =
-          invoke(command, parsedState, adaptedContext);
-      // TODO we used to support accepting ScalaPB and Jackson objects as responses as well.
-      //  Are we OK with losing that? I guess we could have separate methods for that on the
-      //  builder, though it's not obvious how to achieve type-safety for those then.
 
-      // FIXME probably ValueEntityHandler should be adapted to expect an `Effect<? extends
-      // GenerateMessageV3>`?
-      return null;
-    } catch (Exception e) {
-      Throwable unwrapped;
-      if (e.getClass().isAssignableFrom(InvocationTargetException.class) && e.getCause() != null) {
-        unwrapped = e.getCause();
-      } else {
-        unwrapped = e;
-      }
-      if (unwrapped.getClass().isAssignableFrom(FailInvoked$.class)) {
-        throw unwrapped;
-      } else {
-        throw unwrapped.getCause();
-      }
```

Comment:
I don't think we need extra exception handling here. InvocationTargetException we could get if we were doing some reflection and that's exactly what we are not doing. 

Then, if we get `FailInvoked` we just want to rethrow it. 

Finally, in the last case we throw the cause of the exception we got, but not clear what could that exception be and why we are doing so. I suggest that we just don't do any exception handling here.